store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -403,25 +431,26 @@ protected void addAdditionalOperationHandlers() {
                         && !AddElements.class.equals(op)
```

Comment:
Isn't AddElements using a NoOutputHandler?

Reply:
Does this catch ALL other operations? For example, what about the Accumulo Store's GetElementsWithinSet? That does have an output